### DISCUSSION TITLE
httpapp filter

### DISCUSSION THREAD

**Discussion by psyray** [cea-sec/ivre#1403] [Q&A] [answered]
Hi,

Not really an issue but a question, maybe dumb.

What is the `httpapp` filter, and what fills it ?

I don't really understand where the information is taken from the scan to have this information filled in the WebUI

Thanks

**Comment by p-l-** [comment, marked as answer]
This filter matches a "fake" Nmap script (I mean, results that are stored as an Nmap script but really are created from other data) called "http-app".

That fake script is created for some applications: whenever a `microsoftsharepointteamservices` HTTP server header is seen, a "http-app" fake Nmap script is created for MS SharePoint; same for a `kbn-name` header for Kibana.

Also, when specific paths are requested with Zgrab2, we also create such fake scripts (for OWA and Centreon for now).
